bundle/DependencyInjection/Configuration.php

```diff
@@ -42,7 +42,7 @@ public function getConfigTreeBuilder()
                     ->scalarNode('pagelayout')
                         ->info('Default pagelayout used in tag view page')
                         ->cannotBeEmpty()
-                        ->defaultValue('@eZDemo/pagelayout.html.twig')
+                        ->defaultValue('@ezdesign/pagelayout.html.twig')
```

Comment:
I wouldn't use `@ezdesign` here, instead, deprecate the config and use the 'pagelayout' parameter from config directly in the tag view template.

Reply:
Wouldn't it be a problem if some developpers out there are already using the `tag_view.pagelayout` on their own overrides?

And removing it from here, shouldn'it be then also removed from the NetgenTagsExtensions.php & the yml config file too? Or am I missing something?

Reply:
That's why we're deprecating it, and not removing it.

Upgrade instructions will make sure to note that the setting is deprecated and what needs to be done to either keep BC or remove the deprecation notice.

Reply:
Since we can't use https://symfony.com/blog/new-in-symfony-3-4-deprecate-configuration-options  yet to deprecate the configuration option, the best would be to put the deprecation info to the `info()` method call.